java/src/main/java/org/rocksdb/DBOptions.java

```diff
@@ -671,10 +672,17 @@ public DBOptions setDbWriteBufferSize(final long dbWriteBufferSize) {
   public DBOptions setWriteBufferManager(final WriteBufferManager writeBufferManager) {
     assert(isOwningHandle());
     setWriteBufferManager(nativeHandle_, writeBufferManager.nativeHandle_);
+    this.writeBufferManager_ = writeBufferManager;
     return this;
   }
 
   @Override
+  public WriteBufferManager getWriteBufferManager() {
```

Comment:
This should be just `writeBufferManager()` to confirm to the naming pattern of other getters.

Reply:
done!